src/app/visit/status.cpp

```diff
@@ -0,0 +1,20 @@
+#include "status.hpp"
+
+std::string toString(Status status)
+{
+    switch (status)
+    {
+    case Status::Canceled:
+        return "Canceled";
+
+    case Status::Confirmed:
+        return "Confirmed";
+
+    case Status::Relocated:
+        return "Relocated";
+
+    case Status::Visited:
+        return "Visited";
+    }
+    return "Other";
```

Comment:
Let's add "Unassigned" to enum class and return it here as default value.
"Other" does not mean anything

Reply:
Done